private_repo/ansible/inventory/dev/DataPipeline/common.yml

```diff
@@ -5,15 +5,103 @@ domain_name: ""          # your domain name like example.com
 # docker hub details
 dockerhub: "change.docker.url"                                      # docker hub username or url incase of private registry
 private_ingressgateway_ip: ""                        # your private kubernetes load balancer ip
-# Note - You can use the same azure account for the below variables or have separate azure accounts
-sunbird_private_storage_account_name: "change.azure.storage.account.name"         # Azure account name for storing private data (like reports, telemetry data)
-sunbird_public_storage_account_name: "change.azure.storage.account.name"         # Azure account name for storing public data (like contents)
-sunbird_druid_storage_account_name: "change.azure.storage.account.name"         # Azure account name for storing druid data (like query results)
-sunbird_artifact_storage_account_name: "change.azure.storage.account.name"       # Azure account name for storing artifacts data (like jenkins build zip files)
-sunbird_management_storage_account_name: "change.azure.storage.account.name"     # Azure account name for storing backup data (like cassandra backups)
 
+# Cloud Service Provider Variables
+# If cloud_service_provider is AWS then update with access key as value
+# Example: cloud_public_storage_accountname: "AKSHKSJHFJDHJDSHFKSD"
+# If cloud_service_provider is gloud(GCP) then update with service account name
+# Example: cloud_public_storage_accountname: "cloudstorage-gcp-test.iam.gserviceaccount.com"
+# If cloud_service_provider is AZURE then update with stoage account name
+# Example: cloud_service_provider: "azurestotageaccount"
+cloud_public_storage_accountname: ""
+# If cloud_service_provider is AWS then update with region
+# Example: cloud_public_storage_region: us-east-1
+cloud_public_storage_region: ""
+# If cloud_service_provider is gcp then update this variable with project id
+# Example: cloud_public_storage_project: "sunbird-gcp-test"
+cloud_public_storage_project: ""
+# If cloud_service_provider is oci update this variable with namespace
+# Example: cloud_public_storage_namespace: "apsjfhudfjs"
+cloud_public_storage_namespace: ""
 
 
+# Create object storage for each below mentioned variables and update accordingly
```

Comment:
Do we need every bucket mentioned in `Core/common.yml` here also? Or did we just do a copy paste due to missing info on what buckets are exactly needed for this inventory module? If we know the buckets are not needed, we should not define them here. For example, `cloud_storage_playercdn_bucketname` is definitely not required here.